### ISSUE TITLE
DEPRECATION warning from trollop dependency 

### ISSUE THREAD

**Issue by lacostenycoder** [dougbarth/jsonlint#35]
When running `jsonlint` from the command line, it throws:

[DEPRECATION] This gem has been renamed to optimist and will no longer be supported. Please switch to optimist as soon as possible.

I suppose it means to use https://github.com/ManageIQ/optimist

I tested each of these dependencies and just simply `require 'trollop'` gave the deprecation warning.

```
gem dependency jsonlint
Gem jsonlint-0.2.0
  aruba (>= 0, development)
  bundler (~> 1.6, development)
  oj (~> 2)
  rake (>= 0, development)
  rspec (>= 0, development)
  trollop (~> 2)
```

**Comment by dougbarth** [comment]
Thanks for the fix in https://github.com/dougbarth/jsonlint/pull/36

**Comment by szemkhel** [comment]
Do we know when this fix is expected to be in release?

**Comment by dougbarth** [comment]
I just released 0.3.0, which includes this fix.

**Comment by szemkhel** [comment]
That was 🚀 fast. Thank you!
